tests/phpunit/CRM/MembershipExtras/Page/InstalmentScheduleTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+use CRM_MembershipExtras_Test_Fabricator_MembershipType as MembershipTypeFabricator;
+use CRM_MembershipExtras_Test_Fabricator_PriceSet as PriceSetFabricator;
+use CRM_MembershipExtras_Test_Fabricator_PriceField as PriceFieldFabricator;
+use CRM_MembershipExtras_Test_Fabricator_PriceFieldValue as PriceFieldValueFabricator;
+
+/**
+ * Class CRM_MembershipExtras_Page_InstalmentScheduleTest
+ *
+ * @group headless
+ */
+class CRM_MembershipExtras_Page_InstalmentScheduleTest extends BaseHeadlessTest {
+
+  use CRM_MembershipExtras_Test_Helper_FinancialAccountTrait;
+  use CRM_MembershipExtras_Test_Helper_FixedPeriodMembershipTypeSettingsTrait;
+
+  public function testRunRollingMembershipType() {
+    $memType = $this->mockFixedMembershipType('rolling');
+    $today = new DateTime('today');
+    $_REQUEST['schedule'] = 'quarterly';
+    $_REQUEST['start_date'] = $today->format('Y-m-d');
+    $_REQUEST['join_date'] = $today->format('Y-m-d');
+    $_REQUEST['membership_type_id'] = $memType['id'];
+    $_REQUEST['snippet']  = 'json';
```

Comment:
Should we unset the new `$_REQUEST` array keys or use something like [GlobalStackTest.php#L38](https://github.com/civicrm/civicrm-core/blob/9099cab32034e86e611f8e52c4a633494e26eac5/tests/phpunit/CRM/Utils/GlobalStackTest.php#L38) ?
I found it problematic when some tests alter global values - see the second point under "Technical Details" for  https://github.com/compucorp/uk.co.compucorp.membershipextras/pull/362 PR ?.

Reply:
This is bad, phpunit should clean up for us.